CHANGELOG.md

```diff
@@ -1,38 +1,56 @@
-# Changelog
+# CHANGELOG - Terraform MongoDB Atlas Provider 
 
-## [v1.8.1](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.8.1) (March 02, 2023)
+## [v1.8.1](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.8.1) (2023-3-7)
 
-[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.8.1-pre2...v1.8.1)
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.8.0...v1.8.1)
 
-**Merged pull requests:**
-
-- Chore\(deps\): Bump github.com/hashicorp/hcl/v2 from 2.16.0 to 2.16.1 [\#1062](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1062) ([dependabot[bot]](https://github.com/apps/dependabot))
-- Update access\_list\_api\_key.html.markdown [\#1058](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1058) ([Zuhairahmed](https://github.com/Zuhairahmed))
-- Chore\(deps\): Bump github.com/hashicorp/hcl/v2 from 2.15.0 to 2.16.0 [\#1055](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1055) ([dependabot[bot]](https://github.com/apps/dependabot))
-- Chore\(deps\): Bump github.com/gruntwork-io/terratest from 0.41.9 to 0.41.10 [\#1051](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1051) ([dependabot[bot]](https://github.com/apps/dependabot))
-- Update CODEOWNERS to use APIx-Integrations [\#1050](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1050) ([andreaangiolillo](https://github.com/andreaangiolillo))
-- Chore\(deps\): Bump golangci/golangci-lint-action from 3.3.1 to 3.4.0 [\#1026](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1026) ([dependabot[bot]](https://github.com/apps/dependabot))
-
-## [v1.8.1-pre2](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.8.1-pre2) (February 22, 2023)
-
-[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.8.1-pre1...v1.8.1-pre2)
-
-**Closed issues:**
+**Enhancements:**
 
-- Warning: Deprecated attribute [\#1042](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1042)
-- GCP Network Peering remains pending when created via terraform [\#917](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/917)
+- Upgrade to go1.19 [\#1031](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1031) - INTMDB-390
+- Add example for NVME Upgrade [\#1037](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1037) - INTMDB-32
```

Comment:
Add examples to?   I order to figure this out I have to go to the PR to see it's for using mongodbatlas_cluster.  Perhaps add the impacted resource/data source at least to each line where it might add some clarity as to what this is impacting.

Reply:
Sure added links to both examples themselves and links to docs as well whenever a specific resource is mentioned to help with readability